src/custom/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -1,9 +1,85 @@
 import React from 'react'
-import { CurrencyAmount } from '@uniswap/sdk'
-
+import styled from 'styled-components'
+import { Currency, CurrencyAmount } from '@uniswap/sdk'
 import { LONG_PRECISION } from 'constants/index'
+import CurrencyListMod, { StyledBalanceText, Tag as TagMod, TagContainer } from './CurrencyListMod'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { TagInfo, WrappedTokenInfo } from 'state/lists/hooks'
+import { Link } from 'react-router-dom'
+
+const UNSUPPORTED_TOKEN_TAG = [
+  {
+    name: 'Unsupported',
+    description:
+      'This token is unsupported as it does not operate efficiently with the Protocol. Please refer to the FAQ link for more information.',
```

Comment:
I would shorten it a little bit:

```suggestion
      'This token is unsupported as it does not operate efficiently with Gnosis Protocol. Please refer to the FAQ for more information.',
```

Reply:
Although, we made a point to remove references to GP so people aren't confused.
Maybe we should replace GP with CowSwap.

cc @avsavsavs @annamsgeorge

Reply:
I think we should keep the "Gnosis Protocol" so that people start to relate cowswap to gnosis and vice-versa.

Reply:
please give me a final word here or via slack when possible or I will just pick one. happy to go with Gnosis Protocol